benchmarks/operator_benchmark/pt/qactivation_test.py

```diff
@@ -81,8 +90,10 @@ def init(self, dims, permute_dims, inplace, dtype):
         self.set_module_name("QReLU6")
 
 
-op_bench.generate_pt_test(qactivation_configs, QReLUBenchmark)
-op_bench.generate_pt_test(qactivation_configs, QReLU6Benchmark)
+op_bench.generate_pt_test(qactivation_short_configs, QReLUBenchmark)
+op_bench.generate_pt_test(qactivation_short_configs, QReLU6Benchmark)
+op_bench.generate_pt_test(qactivation_long_configs, QReLUBenchmark)
```

Comment:
Configs can be concatenated. So do like this: op_bench.generate_pt_test(qactivation_short_configs + qactivation_long_configs, QReLUBenchmark)